### PR TITLE
Add back machine-controller alerts

### PR DESCRIPTION
**Pull request by metalmatze**
**What this PR does / why we need it**:
We're currently missing the Machine Controller alerts.
These are the same as previously with the new alert format.

Release note:
```release-note
NONE
```

### CODE REVIEW

**Review by alvaroaleman — DISMISSED** [review]

File: config/monitoring/prometheus/rules/rules.yaml [hunk 1]
```diff
@@ -301,6 +313,15 @@ groups:
     for: 15m
     labels:
       severity: critical
+  - alert: MachineControllerDown
+    annotations:
+      message: MachineController has disappeared from Prometheus target discovery.
+      runbook_url: https://github.com/kubernetes-monitoring/kubernetes-mixin/tree/master/runbook.md#alert-name-machinecontrollerdown
```
Comment (mrIncompetent) [inline config/monitoring/prometheus/rules/rules.yaml:361]: ?

File: config/monitoring/prometheus/rules/rules.yaml [hunk 2]
```diff
@@ -236,6 +236,18 @@ groups:
         node_namespace_pod:kube_pod_info:
       )
     record: node:node_net_saturation:sum_irate
+- name: machine-controller
+  rules:
+  - alert: MachineControllerTooManyErrors
+    annotations:
+      message: Machine Controller in {{ $labels.namespace }} has too many errors in
+        its loop.
+      runbook_url: https://github.com/kubernetes-monitoring/kubernetes-mixin/tree/master/runbook.md#alert-name-machinecontrollertoomanyerrors
```
Comment (mrIncompetent) [inline config/monitoring/prometheus/rules/rules.yaml:268]: ?

**Comment by metalmatze** [comment]
@mrIncompetent 
The runbook URLs are going to change and I'll port those descriptions to our docs.

**Comment by metalmatze** [comment]
See issue #1439

**Review by mrIncompetent — APPROVED** [review]

**Pull request merged**